evalml/tests/component_tests/test_components.py

```diff
@@ -849,8 +857,7 @@ def test_no_fitting_required_components(X_y_binary, test_estimator_needs_fitting
                 component.transform(X, y)
 
 
-def test_serialization(X_y_binary, tmpdir, helper_functions):
-    X, y = X_y_binary
+def test_serialization(X_y_binary, ts_data, tmpdir, helper_functions):
```

Comment:
I think we should also add prophet to `test_describe_component()` to make sure the name and params appear as expected